src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -50,6 +50,7 @@
 	private boolean replacing = false;
 	private boolean addon = false;
 	private String permission = "";
+	private String[] permissionMessage = {"&4&lLOCKED", "", "&rYou do not have Permission", "&rto access this Item"};
```

Comment:
Should rather be "noPermissionTooltip" as permissionMessage is misleading...

Reply:
Alright